tools/wptrunner/wptrunner/tests/test_wpttest.py

```diff
@@ -204,13 +206,15 @@ def test_metadata_fuzzy():
                    references=[["/a/fuzzy-ref.html", "=="]],
                    fuzzy=[[["/a/fuzzy.html", '/a/fuzzy-ref.html', '=='],
                            [[2, 3], [10, 15]]]])
-    s = Mock(rel_path="a/fuzzy.html", rel_path_parts=("a", "fuzzy.html"), hash="0"*40)
-    s.manifest_items = Mock(return_value=(item.item_type, [item]))
+    s = mock.Mock(rel_path="a/fuzzy.html", rel_path_parts=("a", "fuzzy.html"), hash="0"*40)
+    s.manifest_items = mock.Mock(return_value=(item.item_type, [item]))
 
 
-    manifest = wptmanifest.Manifest()
+    manifest = wptmanifest.Manifest("")
```

Comment:
Not sure if/what I'm missing, but `Manifest` expects a text string, right? Does `""` have the `Text` type in py2, or does mypy simply not process unit tests?